### PR TITLE
Landtransfer

### PR DESCRIPTION
**Pull request by Denpeer**
Agent is able to build and sell land according to his budget.

User story 1:
As a virtual human I want to be able to sell land to increase my
budget.
User story 2:
As a virtual human I want to be able to buy land, so that I can build
more student housing.

### CODE REVIEW

**Comment by MarcianoJ** [comment]
Looks good, I approve.

File: src/DUWOagent/noBudgetMod.mod2g [hunk 1]
```diff
@@ -8,11 +8,8 @@ exit=always.
...
 	
-	% Build student housing when have it as goal.
-	if bel(functions(FS), 
-		member([Name,Id,L],FS),
-		member('STUDENT',L),
-		getPolygon(10,10,200,20,Polygon)
-		), goal(goalBuildStudentHousing)  
-			then building_plan_construction(Id, 1, Polygon).
+	% Sell our least important piece of land to any stakeholder who will buy it
+	forall bel(stakeholder(BuyerId,_),
```
Comment (klikooo) [inline src/DUWOagent/noBudgetMod.mod2g:14]: Make sure we cant sell to ourself

Comment (Denpeer) [inline src/DUWOagent/noBudgetMod.mod2g:14]: Yes!

**Comment by klikooo** [comment]
Okay looks good now! I approve if travis passes

**Comment by Denpeer** [comment]
More tests will be added with the completion of issue 64 on eishub/tygron

**Comment by MaxGroenenboom** [comment]
It looks good, I approve.

**Pull request merged**